### PR TITLE
[TSC]: Removal of the broken link to approach.md

### LINKED ISSUE #2073: [TSC] Approach link is broken
approach.md was removed, but doc/common still refers to it. This reference should be removed.

### PR DESCRIPTION
**Pull request by CsatariGergely**
closes #2073

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Review by tomkivlin — APPROVED** [review]

File: doc/common/README.md [hunk 1]
```diff
@@ -22,10 +22,6 @@
 
 Cloud iNfrastructure Telco Taskforce [Introduction](chapter00.md). 
 
-<a name="4.0"></a>
```
Comment (pgoyal01) [inline doc/common/README.md:25]: "Approach" needs to be removed from ToC

File: doc/common/README.md [hunk 2]
```diff
@@ -22,10 +22,6 @@
...
-## Approach
-Different specification types within CNTT are related to each other and requires alignments between different parts. The Approach CNTT is taking to tackle those issues is described in [here](./approach.md).
-
 <a name="5.0"></a>
 ## Use Cases
 CNTT Addresses wide range of use cases from Core all the way to the Edge of the Network. Different use cases supported by CNTT specifications are described in [here](./usecases.md).
```
Comment (pgoyal01) [inline doc/common/README.md:27]: nits
```suggestion
The CNTT specifications address a wide range of use cases from the Core all the way to the Edge of the Network. The different use cases supported by the CNTT specifications are described [here](./usecases.md).
```

Comment (ulikleber) [reply doc/common/README.md:27]: Maybe in addition replace CNTT by Anuket.

**Review by ulikleber — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

File: doc/common/README.md (hunk 2 above)
Comment (collivier) [reply doc/common/README.md:27]: I would suggest to that once Anuket is announced and at the beginning of a new release cycle.

**Comment by collivier** [comment]
Could we take Pankaj's comment into account? They seem good.

**Pull request merged**